envoy/access_log/access_log.h

```diff
@@ -55,6 +56,34 @@ class AccessLogManager {
 
 using AccessLogManagerPtr = std::unique_ptr<AccessLogManager>;
 
+struct AccessLogTypeStringValues {
+  const std::string NotSet = EMPTY_STRING;
```

Comment:
`-` would be better than empty string because it's wided used.

Reply:
Used "NotSet" instead, as per your other comment